### PR TITLE
[v1] Delete sprout-generated AST; remove `v1` from AST package path

### PR DESCRIPTION
**Pull request by alancai98**
## Relevant Issues
- https://github.com/partiql/partiql-lang-kotlin/issues/1610

## Description
- Deletes sprout-generated AST
- Remove `v1` from AST package path
- Moves Kotlin sources to Kotlin directory

## Other Information
- Updated Unreleased Section in CHANGELOG: **[NO]**
  - No on v1

- Any backward-incompatible changes? **[YES]**
  - Yes but on v1

- Any new external dependencies? **[NO]**

- Do your changes comply with the [Contributing Guidelines](https://github.com/partiql/partiql-lang-kotlin/blob/main/CONTRIBUTING.md)
  and [Code Style Guidelines](https://github.com/partiql/partiql-lang-kotlin/blob/main/CODE_STYLE.md)? **[NO]**

## License Information

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: partiql-ast/build.gradle.kts [hunk 1]
```diff
@@ -55,24 +55,3 @@ publish {
     name = "PartiQL AST"
     description = "PartiQL's Abstract Syntax Tree"
 }
-
-val generate = tasks.register<Exec>("generate") {
```
Comment (alancai98) [inline partiql-ast/build.gradle.kts:59]: (self-review) remove sprout-generation from AST build

File: partiql-ast/src/main/java/org/partiql/ast/ExcludeStep.java [hunk 2]
```diff
@@ -69,6 +69,8 @@ public <R, C> R accept(@NotNull AstVisitor<R, C> visitor, C ctx) {
     @Builder(builderClassName = "Builder")
     @EqualsAndHashCode(callSuper = false)
     public static class StructWildcard extends ExcludeStep {
+        public StructWildcard() {}
```
Comment (alancai98) [inline partiql-ast/src/main/java/org/partiql/ast/ExcludeStep.java:72]: (self-review) public constructors were missing fro `ExcludeStep.StructWildcard` and `ExcludeStep.CollWildcard`.

File: partiql-cli/src/main/kotlin/org/partiql/cli/format/ast/AstPrinter.kt [hunk 3]
```diff
@@ -1,75 +0,0 @@
...
-import org.partiql.ast.visitor.AstBaseVisitor
-import kotlin.reflect.KVisibility
-import kotlin.reflect.full.isSubclassOf
-import kotlin.reflect.full.memberProperties
-import kotlin.reflect.jvm.jvmErasure
-
-/**
- * Basic printer for debugging during early development lifecycle
- */
-internal object AstPrinter {
```
Comment (alancai98) [inline partiql-cli/src/main/kotlin/org/partiql/cli/format/ast/AstPrinter.kt:12]: (self-review) this CLI object used the sprout-generated AST but it was unused.

File: partiql-ast/build.gradle.kts (hunk 1 above)
Comment (RCHowell) [reply partiql-ast/build.gradle.kts:59]: This is huge, thank you so much!

**Review by RCHowell — APPROVED** [review]

**Pull request merged**